src/mumble/ServerHandler.cpp

```diff
@@ -816,9 +816,20 @@ void ServerHandler::requestUserStats(unsigned int uiSession, bool statsOnly) {
 }
 
 void ServerHandler::joinChannel(unsigned int uiSession, unsigned int channel) {
+	static const QStringList EMPTY;
+
+	joinChannel(uiSession, channel, EMPTY);
+}
+
+void ServerHandler::joinChannel(unsigned int uiSession, unsigned int channel, const QStringList &temporaryAccessTokens) {
 	MumbleProto::UserState mpus;
 	mpus.set_session(uiSession);
 	mpus.set_channel_id(channel);
+
+	foreach(QString tmpToken, temporaryAccessTokens) {
```

Comment:
`for (const auto &tmpToken : temporaryAccessTokens) {`

Reply:
Since `temporaryAccessTokens` is declared as `const`, omitting the specifier in the `for()` loop is okay.